cli/azd/pkg/project/framework_service_python.go

```diff
@@ -147,25 +147,20 @@ func (pp *pythonProject) Package(
 
 const cVenvConfigFileName = "pyvenv.cfg"
 
-func excludeVirtualEnv(path string, file os.FileInfo) bool {
-	if !file.IsDir() {
-		return false
-	}
-
+func isPythonVirtualEnv(path string) bool {
 	// check if `pyvenv.cfg` is within the folder
 	if _, err := os.Stat(filepath.Join(path, cVenvConfigFileName)); err == nil {
```

Comment:
Not a huge fan of silent error conditions. If we end up failing to `stat` (FS failure), the operation doesn't fail, and instead we get the behavior of `false`